### ISSUE TITLE
Help me for run the training via GPU

### ISSUE THREAD

**Issue by SonixLegend** [evanWang1409/FashionNet-Pytorch#1]
I changed your code which can be run with using CUDA. But I found the fasNet model will use 6G GPU memory, and after running loss backward, the memory usage will be expanded to 13G. Then it can't allocate more memory for optimizer step that will need 3G memory. Can you help reduce the model memory size or any other ways? Thanks.
